Dockerfile

```diff
@@ -1,13 +1,20 @@
-FROM busybox:latest
-MAINTAINER Matt Kemp <matt@mattikus.com>
+FROM busybox:latest AS builder
+LABEL MAINTAINER="Matt Kemp <matt@mattikus.com>"
 
 ENV version=1.3.2
 
 # Download statically compiled murmur and install it to /opt/murmur
-ADD https://github.com/mumble-voip/mumble/releases/download/${version}/murmur-static_x86-${version}.tar.bz2 /opt/
-RUN bzcat /opt/murmur-static_x86-${version}.tar.bz2 | tar -x -C /opt -f - && \
```

Comment:
If we're using a multi stage build, do we get a benefit from compression the layers here? Is wget preferred over ADD these days? I haven't kept up as much.

Reply:
`ADD` is not recommended due to its ambiguous behavior. For example with [compressed files](https://stackoverflow.com/a/24958548/4172336) it might extract a `.tar.bz2` if it's done locally, but won't if it's done remotely. 
How it outputs can be unexpected as well. `/opt/` writes it to the directory `/opt/`. However, `/opt` would write to a file at `/opt`
So, using wget is more clear in its intent. 

[Best Practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy)
> Because image size matters, using ADD to fetch packages from remote URLs is strongly discouraged; you should use curl or wget instead. That way you can delete the files you no longer need after they’ve been extracted and you don’t have to add another layer in your image. 

There is no file-size benefit from compressing this layer when using multi-stage.

Reply:
Awesome, thanks for the pointers!